blazor/accumulation-chart/legend.md

```diff
@@ -293,6 +293,47 @@ When the legend items exceed legend bounds, paging will be enabled by default. E
 
 ![Paging for Legend in Blazor Accumulation Chart](images/legend/blazor-accumulation-chart-legend-paging.png)
 
+### Legend Paging Customization
```

Comment:
Change the header to ## Paging Customization instead of Legend Paging Customization

Reply:
Header Changed